monkestation/code/modules/antagonists/mimic/mimic.dm

```diff
@@ -0,0 +1,561 @@
+#define MIMIC_HEALTH_FLEE_AMOUNT 30
+#define MIMIC_JITTER_CHANCE 0.8
+#define MIMIC_DISGUISE_COOLDOWN 5 SECONDS
+
+/mob/living/simple_animal/hostile/alien_mimic
+	name = "mimic"
+	real_name = "mimic"
+	desc = "A morphing mass of black gooey tendrils."
+	speak_emote = list("warbles")
+	emote_hear = list("warbles")
+	faction = list("aliens") //don't wanna have them attack eachother
+	icon = 'monkestation/icons/mob/animal.dmi'
+	icon_state = "mimic"
+	icon_living = "mimic"
+	icon_dead = "mimic_dead"
+	move_to_delay = 0.5
+	var/disguised_move_delay = 4
+	a_intent = INTENT_HARM
+	stop_automated_movement = 1
+	status_flags = CANPUSH
+	stat_attack = UNCONSCIOUS
+	mob_size = MOB_SIZE_SMALL
+	pass_flags = PASSTABLE | PASSMOB
+	unsuitable_atmos_damage = 0 //They won't die in Space!
+	minbodytemp = TCMB
+	maxbodytemp = T0C + 40
+	maxHealth = 90
+	health = 90
```

Comment:
Boost them to 120, I think a 25% increase will be fair based off the playtesting we have done.